tpm2/tpm2.go

```diff
@@ -1205,17 +1205,22 @@ func NVDefineSpace(rw io.ReadWriter, owner, handle tpmutil.Handle, ownerAuth, au
 		AuthPolicy: policy,
 		DataSize:   dataSize,
 	}
-	return NVDefineSpaceEx(rw, owner, ownerAuth, authString, nvPub)
+	authArea := AuthCommand{
+		Session:    HandlePasswordSession,
+		Attributes: AttrContinueSession,
+		Auth:       []byte(ownerAuth),
+	}
+	return NVDefineSpaceEx(rw, owner, authString, nvPub, authArea)
 
 }
 
-// NVDefineSpaceEx accepts NVPublic structure and is more flexible.
-func NVDefineSpaceEx(rw io.ReadWriter, owner tpmutil.Handle, ownerAuth, authVal string, pubInfo NVPublic) error {
+// NVDefineSpaceEx accepts NVPublic structure and AuthCommand. Allows more flexibility
```

Comment:
nit: Go comments are meant to be complete English sentences, can you rephrase to something like:

`// NVDefineSpaceEx accepts NVPublic structure and AuthCommand, allowing more flexibility.`
`// NVDefineSpaceEx accepts NVPublic structure and AuthCommand. This allows more flexibility.`